src/app/zap-templates/zcl/data-model/chip/test-cluster.xml

```diff
@@ -155,5 +155,18 @@ limitations under the License.
       <arg name="returnValue" type="INT8U"/>
     </command>
 
+    <command source="server" code="0x02" name="TestAddArrayofStructArguments" optional="false">
+      <description>
+        Command that takes a array of struct arg
+      </description>
+      <arg name="arg1" type="SimpleStruct" array="true"/>
+    </command>
+
+    <command source="server" code="0x03" name="TestAddNestedArrayArguments" optional="false">
```

Comment:
Again, what's the "Add" here?  And again, why is source "server"?

Reply:
it has been moved in #10171